app/tests/challenges_tests/test_forms.py

```diff
@@ -15,17 +14,45 @@
 )
 
 
+@pytest.mark.parametrize(
+    "required_field",

```

Comment:
Isn't this going to cause a slowdown of the tests as you have to repeat the setup each time? Seems like this would be easy to check in a single test.